index.html

```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession2">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession3">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
+                    </div>
+                </div>
+            
+            </div>
+        </div>
     </div>
-    <button class="second-btn" type="button">See Project</button>
-  </div>
+      
 </section>
 <!--about me starts -->
  <section id="about">
     <div class="about-img">
-    <h3 class="me">About me</h3>
-    <p class="info">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-    <div class="resume">
-        <button class="last-btn" type="button">Get My Resume</button>
-    </div>
-    <hr class="second-line">
+        <div class="styl">
+            <h3 class="me">About me</h3>
+                <p class="info">Hello I’m a software developer! I can help you build a<br> product , feature or website Look through some of my work<br> and experience! If you like what you see and have a project<br> you need coded, don’t hestiate to contact me.
+                </p>
+            <div class="resume">
+                <button class="last-btn" type="button">Get My Resume</button>
+            </div>
+        </div>
+        <div class="line-cont">
+            <hr class="second-line">
+        </div>  
```

Comment:
- [ ] kindly increase the width of the `About me` div to match the figma design

![image](https://user-images.githubusercontent.com/89282221/188262996-7005e0b1-b34f-4b75-881f-6385e584d464.png)